projects/kit/components/rating/rating.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    forwardRef,
+    HostBinding,
+    Inject,
+    Input,
+    Optional,
+    Self,
+} from '@angular/core';
+import {NgControl} from '@angular/forms';
+import {AbstractTuiControl, TUI_FOCUSABLE_ITEM_ACCESSOR} from '@taiga-ui/cdk';
+
+import {RatingOptions, TUI_RATING_OPTIONS} from './rating.options';
+import {roundRatingBy} from './utils/round-rating-by';
+
+@Component({
+    selector: 'tui-rating',
+    templateUrl: './rating.template.html',
+    styleUrls: ['./rating.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: TUI_FOCUSABLE_ITEM_ACCESSOR,
+            useExisting: forwardRef(() => TuiRatingComponent),
+        },
+    ],
+})
+export class TuiRatingComponent extends AbstractTuiControl<number> {
+    @Input() max: number = this.options.max;
+    @Input() iconNormal: string = this.options.iconNormal;
+    @Input() iconFilled: string = this.options.iconFilled;
+    focused: boolean = true;
```

Comment:
Why is it true by default? And why do we need it?

Reply:
fixed